regression/strings-smoke-tests/java_parselong_decimal_2/test.desc

```diff
@@ -1,6 +1,6 @@
 THOROUGH
-test_parselong.class
---refine-strings
+Test.class
```

Comment:
I checked all the tests and switched those that took less than 3 seconds to CORE